src/mds/DamageTable.h

```diff
@@ -20,8 +20,7 @@
 
 #include "mdstypes.h"
 #include "include/random.h"
-
-class CDir;
+#include "CDir.h"
```

Comment:
Why was this change necessary?

Reply:
oops! My bad. That was just a pointer. Please revert it.